cpp/unreal_plugins/OpenBot/Source/OpenBot/OpenBot.Build.cs

```diff
@@ -11,15 +11,17 @@ public class OpenBot : ModuleRules
     public OpenBot(ReadOnlyTargetRules Target) : base(Target)
     {
         // Disable precompiled headers (in our code but not Unreal code) for faster builds,
-        // easier debugging of compile errors, and strict enforcement of include-what-you-use
+        // easier debugging of compile errors, and strict enforcement of include-what-you-use.
         PCHUsage = ModuleRules.PCHUsageMode.Default;
         PrivatePCHHeaderFile = "";
         bUseUnity = false;
 
-        // Turn off code optimization except in shipping builds for faster build times
+        // Turn off code optimization except in shipping builds for faster build times.
         OptimizeCode = ModuleRules.CodeOptimization.InShippingBuildsOnly;
 
-        // Enable exceptions because some of our third-party dependencies use them
+        // Our ASSERT macro throws exceptions, and so does our templated function Config::get(...),
+        // because it depends yaml-cpp, which throws exceptions. So we need to enable exceptions
```

Comment:
'it depends yaml-cpp' -> 'it depends `on` yaml-cpp'

Reply:
Good catch.